ow_lander/scripts/arm_action_servers.py

```diff
@@ -793,6 +796,86 @@ def now_from_start(start):
             self._server.set_aborted(self._result)
 
 
+class ARM_MOVE_JOINT(object):
```

Comment:
I think we should stick to our Python class name convention.
```suggestion
class ArmMoveJoint(object):
```